files/install_graphdb.sh

```diff
@@ -112,6 +112,13 @@ echo 'fs.file-max = 262144' | tee -a /etc/sysctl.conf
 
 sysctl -p
 
+echo "##################################################"
+echo "#    Setting Azure Application Insights agent    #"
+echo "##################################################"
+
+wget -P /opt/graphdb/ https://github.com/microsoft/ApplicationInsights-Java/releases/download/3.4.19/applicationinsights-agent-3.4.19.jar
```

Comment:
Not sure about placing it in /opt/graphdb. @yaskoo, is there a proper linux place for this?

Reply:
I think the "linux" rules are more strict if you install with a package manager, otherwise "all-in-opt" should be fine.